linkerd/app/outbound/src/tests/mod.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{
+    endpoint::{TcpConcrete, TcpLogical},
+    Config,
+};
+use futures::prelude::*;
+use ipnet::IpNet;
+use linkerd2_app_core::{
+    config, exp_backoff, profiles,
+    proxy::http::h2,
+    svc::NewService,
+    transport::{listen, tls},
+    Error, IpMatch,
+};
+use linkerd2_app_test as test_support;
+use std::{net::SocketAddr, str::FromStr, time::Duration};
+use tower::ServiceExt;
+
+mod tcp;
+
+const LOCALHOST: [u8; 4] = [127, 0, 0, 1];
+
+fn profile() -> profiles::Receiver {
```

Comment:
nit: `default_profile()`? since this doesn't give you any control over which profile is used

Reply:
this code was all already there, i just moved it from `tests.rs` into `tests/mod.rs`